docs/notebooks/JAX_for_the_impatient.ipynb

```diff
@@ -0,0 +1,1293 @@
+{
```

Comment:
<p>Maybe add "(there is no special meaning to the two unsigned int32s -- it's just a way of representing a uint64)" [I <em>think</em> this is true]</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google/flax/pull/542/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Reply:
<p>I'll trust you on that. At least I didn't read otherwise anywhere! Added</p>